packages/app-i18n-content/package.json

```diff
@@ -42,11 +41,6 @@
     "build": "yarn webiny run build",
     "watch": "yarn webiny run watch"
   },
-  "svgo": {
```

Comment:
BTW just a confirmation, the `svgo` is no longer needed right?

Reply:
It is, but I moved `svgo` configuration from separate `package.json` files to `svgr` loader config in webpack to be more consistent.